api/src/main/java/org/openmrs/ui/framework/UiUtils.java

```diff
@@ -397,7 +397,20 @@ public String formatDatePretty(Date date) {
 			return format(dateExt.getDateWithoutTime());
 		}
 	}
-	
+
+	/**
+	 * Formats a date without any timezone conversion
+	 *
+	 * @param date the date to format
+	 * @return date formatted
+	 */
+	public String formatDateWithoutTimezoneConversion(Date date) {
+		String dateFormatGP = Context.getAdministrationService()
```

Comment:
Shouldn't you just use `getDateFormat()`?

Reply:
I will use that. thank you